### ISSUE TITLE
Function to update text

### ISSUE THREAD

**Issue by jamesbradlee** [FGRibreau/spinners#37]
There is currently no way to update the text in the spinner, would be nice.

**Comment by l0uisgrange** [comment]
Sorry for bumping this, but this would indeed be very useful. Is it planned?

**Comment by FGRibreau** [comment]
Yes, @l0uisgrange @jamesbradlee please send a PR along with tests so we can discuss it and find a way to merge that :)
